src/coreComponents/mesh/MeshObjectPath.cpp

```diff
@@ -123,11 +124,12 @@ MeshObjectPath::fillPathTokens( string const & path,
         existingMeshBodyAndLevel += "/n";
       } );
 
-      GEOSX_ERROR_IF( !levelNameFound,
-                      GEOSX_FMT( "Path ({}) specifies an invalid MeshBody or MeshLevel. ",
-                                 "existing MeshBodies: MeshLevels /n",
+      GEOSX_THROW_IF( !levelNameFound,
+                      GEOSX_FMT( "Path {} specifies an invalid MeshBody or MeshLevel. ",
+                                 "existing MeshBodies: {} /n",
```

Comment:
```suggestion
                                 "existing MeshBodies: {} \n",
```
Is it a legit '/'?

Reply:
Indeed, they weren't legit, I updated that part of the code in the next commit: [updating "invalid MeshBody" error message.](https://github.com/GEOSX/GEOS/pull/2341/commits/930dd64f45bcdcbf8ec1ae3340e4ed78c36d85c8)